### ISSUE TITLE
[python-lessons] Anaconda installer may not be available to all learners

### ISSUE THREAD

**Issue by bsmith89** [swcarpentry/curriculum-advisors#10]
Some community members have suggested that a subset of learners may not have free access to the Anaconda installer that the Carpentries recommends for obtaining Python and most of its packages.

A discussion of this issue with more details is ongoing here: swcarpentry/python-novice-gapminder#620.

**Comment by coatless** [comment]
Yes, this is a problem. We probably want to loop the carpentries legal officer in to verify any change. 

The gist of the issue is starting in 2020 anaconda shifted its Terms of Service to prohibit commercial usage of its default channels. Thus, `Anacodna`, `miniconda`, and even `conda` are problematic to use as distributed with **the `default` channel configuration**. The simple fix is to change/remove `defaults` channel in favor of using `conda-forge`. 

```bash
# View all channels currently set
conda config --show channels

# Remove the problematic defaults channel
conda config --remove channels defaults

# Setup conda-forge as the main channel
conda config --add channels conda-forge
```

Alternatively, we can replace the installation with [`{miniforge}`](https://github.com/conda-forge/miniforge). This distribution of `miniconda` sets [conda-forge](https://conda-forge.org/) as the default (and only) channel to avoid any issues.

**Note:** `Anacodna`, `miniconda`, and `conda` are all released under BSD license. The problem is more the distribution channel.

**Comment by Gina123-P** [comment]
We could recommend pycharm.

**Comment by bsmith89** [comment]
Hmm, can you say more @Gina123-P ? I didn't realize that PyCharm came with an installation of the packages that we need (but, honestly, I haven't used it recently, so I don't know what might have changed).

Googling around, the PyCharm documentation seems to suggest that you need to install Python separately, and they mention conda as one way to do that. Perhaps I'm missing something.

Also, does anyone know what the licensing is like for PyCharm? I know it's a commercial offering; is it more permissive than Anaconda?

**Comment by annajiat** [comment]
Faced this issue while  teaching a workshop last week  where participants informed that they cannot use anaconda. I would like to add that, manually installing and running jupyter required some troubleshooting as it was a common mistake to miss a space or hyphen while typing. In general, a lot of messages  appearing on the screen during the installation was confusing for the learners. This  happened even though a setup document  tailored to the organization’s environment was prepared and shared beforehand. More than half of the learners were able to follow the guidelines.

**Comment by mhagdorn** [comment]
We also cannot use anaconda. I think we would also require licenses for pycharm. So, I think some official guidance of what to use instead would be good.

**Comment by RabeaMue** [comment]
facing the same issue for our workshops. A official guidance of how to use eg miniforge or miniconda would be useful for us as well 👍
